.eslintrc.js

```diff
@@ -4,7 +4,4 @@ module.exports = {
   parserOptions: {
     project: "./tsconfig.eslint.json",
   },
-  rules: {
-    "unicorn/prefer-module": "off",
```

Comment:
this we need, it was there before :P

Reply:
This one was already exported globally

Reply:
Why do we need to remove `prefer-module`? That rule enforces ESM (which is what we want)

Reply:
we were getting errors in this repo, that's why i said that, no other reason

Reply:
That is not reason enough to disable a rule, I can take a look at the errors with you and we can figure out specifically for that repo.

Therefore I will merge it as it is!

Reply:
Not about our repo, but this repo here